crates/bootstrap_srv/src/space.rs

```diff
@@ -0,0 +1,228 @@
+use std::collections::HashMap;
+use std::sync::{Arc, Mutex};
+
+use crate::store::*;
+
+/// A map of spaces.
+#[derive(Clone)]
+pub struct SpaceMap(Arc<Mutex<HashMap<bytes::Bytes, Space>>>);
+
+impl Default for SpaceMap {
+    fn default() -> Self {
+        Self(Arc::new(Mutex::new(HashMap::new())))
+    }
+}
+
+impl SpaceMap {
+    /// Read the content of a space.
+    pub fn read(&self, space: &bytes::Bytes) -> std::io::Result<Vec<u8>> {
```

Comment:
same here - should this be a `SpaceId`?

Reply:
Done: https://github.com/holochain/kitsune2/pull/10/commits/263d5ab3f9e742b19c46ce7a11d370205ad904ae